app/assets/stylesheets/employee-page.scss

```diff
@@ -1,9 +1,5 @@
-strong {
-	font-weight: bold; 
-}
-
 em {
```

Comment:
The file is two line spaces but set the tabs to two line spaces and hope that is fixed for github